velox/exec/Task.cpp

```diff
@@ -2433,6 +2434,14 @@ ContinueFuture Task::taskDeletionFuture() {
   return std::move(future);
 }
 
+std::string Task::printPlanWithStats(bool includeCustomStats) const {
+  if (planFragment_.planNode) {
+    return exec::printPlanWithStats(
+        *planFragment_.planNode, taskStats_, includeCustomStats);
+  }
+  return "";
```

Comment:
Users of the API may be surprised to get an empty string? When would this happen? Should this be an error?